container/Home/Search/Search.style.js

```diff
@@ -83,11 +83,6 @@ export const SearchWrapper = styled.div`
   bottom: 81px;
   z-index: 1;
 
-  @media only screen and (min-width: 569px) and (max-width: 991px) {
```

Comment:
Why was remove this media query?

Reply:
because it affected with responsive design